Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "lightning-sys"
 version = "0.2.1"
-authors = ["Peter Elliott <pelliott@ualberta.ca>"]
+authors = ["Peter Elliott <pelliott@ualberta.ca>", "Darren Kulp <darren@kulp.ch>"]
```

Comment:
I hope adding my name is all right here. I do not care so much, but I thought it would make sense to match the `crates.io` ownership list.

Reply:
About time :^)